api/handler/api/util.go

```diff
@@ -27,19 +27,17 @@ func requestToProto(r *http.Request) (*api.Request, error) {
 		Url:    r.URL.String(),
 	}
 
-	ct, _, err := mime.ParseMediaType(r.Header.Get("Content-Type"))
-	if err != nil {
-		ct = "application/x-www-form-urlencoded"
-		r.Header.Set("Content-Type", ct)
+	if _, ok := r.Header.Get("Content-Type"); ok {
```

Comment:
`Header.Get` returns one variable, not two
https://golang.org/pkg/net/http/#Header.Get

Reply:
strange otherwise VS Code shows me that error, thanks I did not want to build it all again as I have go version 1.12 and did not want to mess up my environment. I'll update that. What about the body part? do you also think to pass it around independent of the Content Type.